### PR TITLE
Fix compiling with vs2019 and sdk extras

### PR DESCRIPTION
**Pull request by miterosan**
Fixes the 
``
'obj\Release\monoandroid81\Resource.Designer.cs' could not be found
``
error.

### CODE REVIEW

File: osu.Framework.Android/osu.Framework.Android.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras/1.6.55">
+﻿<Project Sdk="MSBuild.Sdk.Extras/2.0.41">
```
Comment (swoolcock) [inline osu.Framework.Android/osu.Framework.Android.csproj]: Are you building with .NET Core 3 Preview?  The README suggests that it's required for version 2+
https://github.com/onovotny/MSBuildSdkExtras

> If you are using MsBuild.Sdk.Extras version 2 or above, use .NET Core 3 Preview to compile. You can still target previous versions of .Net Core

Ultimately we should be specifying the Sdk Extras version outside the `csproj` files anyway.

**Comment by peppy** [comment]
Seems to [be fixed](https://github.com/onovotny/MSBuildSdkExtras/issues/181#issuecomment-533302373), please confirm.

**Comment by SoerenDaenekas** [comment]
Is fixed with the newest preview of visual studio 2019. 

Please note that cake does not find the newest location of msbuild yet because they changed the location of msbuild.

**Comment by miterosan** [comment]
Is fixed with the newest preview of visual studio 2019.

Please note that cake does not find the newest location of msbuild yet because they changed the location of msbuild.

**Pull request closed without merging**